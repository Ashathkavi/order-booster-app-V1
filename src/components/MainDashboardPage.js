import React from 'react'
import {Link} from 'react-router-dom'
import moment from 'moment'

const now = moment()



const MainDashboardPage = () => {
    return(
        <div>
            <h4>{now.format('MMM Do, YYYY')}</h4>
            <h3>This from Main dashboard componrnet</h3>
            <p><Link  to="/food"> Manage Food</Link></p>
            <p><Link  to="/order"> Manage Order</Link></p> 
        </div>
    )}

export default MainDashboardPage