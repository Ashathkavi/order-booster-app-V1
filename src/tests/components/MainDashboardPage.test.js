import React from 'react'
import {shallow} from 'enzyme'
import MainDashboardPage from '../../components/MainDashboardPage'

test('should render MainDashboardPage', () => {
    const wrapper = shallow(<MainDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})

