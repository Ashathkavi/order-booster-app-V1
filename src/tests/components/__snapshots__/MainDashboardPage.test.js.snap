// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MainDashboardPage 1`] = `
<div>
  <h4>
    Jan 1st, 1970
  </h4>
  <h3>
    This from Main dashboard componrnet
  </h3>
  <p>
    <Link
      to="/food"
    >
       Manage Food
    </Link>
  </p>
  <p>
    <Link
      to="/order"
    >
       Manage Order
    </Link>
  </p>
</div>
`;
